tools/depends/native/config.buildsteps.in

```diff
@@ -1,5 +1 @@
-export PKG_CONFIG=@prefix@/@tool_dir@/bin/pkg-config
-export YASM=@prefix@/@tool_dir@/bin/yasm
-export NASM=@prefix@/@tool_dir@/bin/yasm
-export CMAKE=@prefix@/@tool_dir@/bin/cmake
 export PATH="@prefix@/@tool_dir@/bin:$PATH"
```

Comment:
@wsnipex is this way OK to change the jenkins `PATH` to make sure the from native generated tools are used first.

Thats also the reason why before on jenkins build it has used on Linux the cmake 2.8.7 from his `/usr/bin` where I thought in native is a old version.